warehouse/query-core/src/test/java/datawave/query/CompositeFunctionsTest.java

```diff
@@ -491,7 +493,7 @@ public void testNotNulls() throws Exception {
                 "filter:isNotNull(UUID) || filter:isNotNull(NULL1)",
                 "filter:isNotNull(ONE_NULL)",
                 "UUID =~ '^[CS].*' AND filter:isNotNull(UUID)",
-                "UUID =~ '^[CS].*' AND filter:isNotNull(NULL1)",
+                "UUID =~ '^[CS].*' AND filter:isNotNull(NULL1)",    //  dis guy
```

Comment:
Could you add a more descriptive comment?

Reply:
That's a leftover comment, I'll remove it